FreeRTOS/FreeRTOSConfig.h

```diff
@@ -69,8 +69,13 @@
 #define configAPPLICATION_ALLOCATED_HEAP        1   
 #endif
 
-__attribute__((section(".sdramMemorySection")))
-static uint8_t ucHeap[configTOTAL_HEAP_SIZE];
+/**
+ * @brief FreeRtos heap declaration
+ * First attribute puts heap into sram
+ * Second suppresses warnings
+ */
+__attribute__((section(".sdramMemorySection")))__attribute__((unused))

```

Comment:
You can have them both in the one `__attribute__`

Reply:
fixed in new commit